views/openacademy_menu.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <!-- ACTIONS OPENING VIEWS ON MODELS -->
+    <record id="action_window_course" model="ir.actions.act_window">
+        <field name="name">Open course form</field>
+        <field name="res_model">openacademy.course</field>
+        <field name="view_mode">form</field>
```

Comment:
`tree,form` was correct, only tre view was missing.

You need a tree (list) view to be able to see all courses and select one. Once one is selected, the form view will be open. If only a form view is defined, you won't be able to see all courses.

So:
- Restore previous view mode
- Create a tre view


The above also applies for the other model